Assets/_BForBoss/_Core/Scripts/Managers/BaseWorldManager.cs

```diff
@@ -40,13 +53,15 @@ protected virtual void Awake()
         {
             _stateManager.OnStateChanged += HandleStateChange;
             _inputSettings = new InputSettings();
+            SceneManager.LoadScene("WeaponManager", LoadSceneMode.Additive);
```

Comment:
i'm assuming this is synchronous? 
So Start() can't be called until this load is completed?

Reply:
So it looks like its synchronous but on the next frame, it doesn't seem to currently be causing any issues though